SU2_CFD/src/driver_structure.cpp

```diff
@@ -853,6 +853,11 @@ void CDriver::Geometrical_Preprocessing() {
       geometry_container[iZone][iInst][MESH_0]->SetControlVolume(config_container[iZone], ALLOCATE);
       geometry_container[iZone][iInst][MESH_0]->SetBoundControlVolume(config_container[iZone], ALLOCATE);
 
+      /*--- Compute the max length. ---*/
+
+      if ((rank == MASTER_NODE) && (!fea)) cout << "Finding max control volume width." << endl;
+      geometry_container[iZone][iInst][MESH_0]->SetMaxLength();
```

Comment:
I was under the impression that the multigrid solver didn't impact the turbulence model at all. Does the turbulence solver use a geometry object other than the one stored in `[MESH_0]`?

Reply:
The turbulence solver does not use MG, that is true. However, does this max length value affect the level of dissipation used in the upwind flux for the mean flow? If yes, then it could be that we want this on all levels for a more consistent flux on the coarse levels (although 1st-order). If this only affects the turbulence solver (for DES or turb. convective scheme), then it is probably not an issue.

Reply:
Hi @clarkpede ,

Thanks for the fix. Can you provide the same Cylinder's figure using the FD low dissipation instead of NTS ?

Eduardo

Reply:
@EduardoMolina Here's a comparison of the Roe blending coefficients using FD belnding and NTS blending:

![fd_nts_comparison](https://user-images.githubusercontent.com/13340225/43531521-910a6e7e-9575-11e8-8139-5af8fae583f1.png)

Unfortunately, this comparison is not as useful as I would have hoped.  That's because the NTS method is not design to be exactly like the FD method; it was designed to overcome some of the weaknesses of the FD method of blending.

The FD method is designed to:

+ Go to 1 in attached boundary layers
+ Go to 0 towards the edge of boundary layers

The NTS formulation was explicitly designed to:

+ Go to 1 in attached boundary layers
+ Go to 1 in irrotational regions, even if strain is present (i.e., the freestream)
+ Go to 0 in regions with high vorticity.

As can be seen in the figure, both methods are serving their original purpose.  Where they differ is the wake region, which is not clearly a boundary layer, nor clearly the freestream.  Because the NTS formulation explicitly separates strain vs. rotation, its behavior in this highly rotational region is different than the FD method.

I'm running a simpler case, with a backwards facing step, right now, in order to compare the two methods on more equal ground.

Reply:
@economon I've added the max length calculations to multigrid.

Reply:
Thanks @clarkpede 

However, I am pretty sure that there is something wrong with the FD method. It seems that you calculated FD as FD=1.0-NTS.... If you look at the figure, it is different from what you correct described as the FD method.

Please see the average of FD method (computed from several thousands time-steps) for an ice-airfoil test case. I computed this using my original version of the code based on SU2 v5.0

![ice_fd](https://user-images.githubusercontent.com/9790985/43681921-4dd0ac46-9817-11e8-97a0-ec2eb8c05c70.png)

Best,

Eduardo

Reply:
@clarkpede Any news regarding these remarks ? @EduardoMolina Does this need to be adressed before merging ?